model_loader.py

```
@@ -0,0 +1,13 @@
+import os
+import tensorflow as tf
+from utils.helper import get_logger
+
+logger = get_logger(__name__)
+
+def load_model(model_path: str = "./models/team12.h5"):
```

Ruff (0.11.9)

Missing return type annotation for public function `load_model`

(ANN201)